source/org/zfin/db/postGmakePostloaddb/1135/ZFIN-8188-remove-unmatched-closing-div-tag.sql

```diff
@@ -0,0 +1,9 @@
+--liquibase formatted sql
+--changeset rtaylor:ZFIN-8188
+
+select zdb_id from publication where pub_abstract ilike '%</div>' and pub_abstract not ilike '%<div%';
```

Comment:
Does this assume that we only encounter </div> with <div> elements but not vice versa? Is that actually true?